tests/fix_course_delete_module_test.php

```diff
@@ -247,4 +250,102 @@ public function find_taskid(\core\task\adhoc_task $task) {
         }
         return $taskid;
     }
+
+    /**
+     * Common set up for diagnoser, reporter, surgeon tests
+     *
+     */
+    public function setup_test() {
```

Comment:
it should define return type
`    public function test_delete_task_setup():void {`